src/test/java/org/jabref/logic/formatter/casechanger/LowerCaseFormatterTest.java

```diff
@@ -19,6 +19,7 @@ public void setUp() {
 
     @Test
     public void test() {
+        assertEquals("lower", formatter.format("lower"));
```

Comment:
Could you maybe change that to a paramterized test? We aim for having one assertion per test case. See also https://github.com/JabRef/jabref/issues/6207

![grafik](https://user-images.githubusercontent.com/1366654/116923413-e5f2ed80-ac56-11eb-9b47-c6eb8072211b.png)